app/models/duplicate.rb

```diff
@@ -1,3 +1,11 @@
 class Duplicate < ApplicationRecord
   belongs_to :duplicatable, polymorphic: true
+
+  after_commit :maybe_notify_duplicates, on: :create
+
+  def maybe_notify_duplicates
```

Comment:
not sure if this is handled but we only want to notify when it's over the threshold, same logic that drives the red badge in the top nav and whether or not the fix duplicates button shows up

Reply:
I'm calling the "duplicates" method it by default only returns those over the lower threshold see here https://github.com/radicalbear/rad_common/blob/4b64b03f1e9aa06513931d850eda7cd405e3490e/app/models/concerns/duplicate_fixable.rb#L130

Reply:
nah, we want the same logic that decides whether or not the badge shows up in the nav, for example see [here](https://github.com/radicalbear/rad_common/blob/main/app/helpers/duplicates_helper.rb#L23)

Reply:
Ok gotcha, I will update that, out of curiosity, whats the use case for the lower threshold, I see it used here 
https://github.com/radicalbear/rad_common/blob/5b132b6f139ddd320b7a4f76521ced8e386742c3/app/presenters/card_presenter.rb#L227

I was just wondering if that logic is either redundant and/or possibly conflicting?

Reply:
I don't see the lower threshold used there, am I missing it?

Reply:
@garyfoster I'm not sure anymore, maybe I was reading something wrong? I think I'm good now